lms/djangoapps/learner_home/test_views.py

```diff
@@ -1066,3 +1071,36 @@ def test_no_enrollable_course(
             response_content.get("courses"),
             self.SERIALIZED_GENERAL_RECOMMENDATIONS,
         )
+
+    @ddt.data(
+        (True, False),
+        (False, True),
+        (False, False),
+        (True, True),
```

Comment:
You can add the expected value in this list as well
```suggestion
        (True, False, None),
        (False, True, False),
        (False, False, None),
        (True, True, True),
```